conda_recipe/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM continuumio/miniconda
+
+RUN apt-get update && apt-get install -y gcc libgl1-mesa-glx
```

Comment:
to keep layers small one adds a `&& rm -rf /var/lib/apt/lists/*` at the end of a apt-get line in a container

(this also has a security plus for people using your container to be forced taking new repos in first via `update` but that's less important here)